plugins/snowflake/dbt/include/snowflake/macros/materializations/helpers.sql

```diff
@@ -0,0 +1,121 @@
+{% macro incremental_validate_on_schema_change(on_schema_change, default_value='ignore') %}
```

Comment:
Are any of the macros in this file different from the ones defined in `global_project`? If not, you don't need to define them again here, Snowflake will use the default versions. (I see `snowflake_diff_columns` below, but I don't actually think it's different from `diff_columns`.)

Reply:
Got it - moving the logic from snowflake to `global_project`.